blockchain/filedao/filedao_header.go

```diff
@@ -23,13 +23,17 @@ type (
 		Compressor     string
 		BlockStoreSize uint64
 		Start          uint64
-		End            uint64
-		TopHash        hash.Hash256
+	}
+
+	// FileTip is tip info of chain
+	FileTip struct {
```

Comment:
only these 2 need to be updated on each block commit

the header should only be written once in the entire lifetime of chain db file (first time Start() after new file is created)